Mapping/ClassMetadata.php

```diff
@@ -106,6 +115,22 @@ public function getIri()
     }
 
     /**
+     * @return string
+     */
+    public function getIdentifier()
+    {
+        return $this->identifier;
```

Comment:
Ok I understand, i didn't realize that doctrine identifier can be set without annotation ... my bad.

* So yes keep a DoctrineIdentifierLoader.
* Add our own annotation in AnnotationLoader. 
* Set the default value by default in ClassMetadata. But for this we need to add a check in AttributeLoader ( if there are no id attribute, the value of classMetadata->getIdentifier must be valid)
* The doctrineLoader must be executed after AttributeLoader but before AnnotationLoader.

Good for you ?